### DISCUSSION TITLE
add white board

### DISCUSSION THREAD

**Discussion by invisiblecoder99** [edumeet/edumeet#838] [Ideas]
can you add white board it will be useful for teaching

**Comment by havfo** [comment]
This feature is currently not very high on our list of priorities, but we will get to it eventually.

**Comment by misi** [comment]
Cisco did this in their webex board products.. So I also think it would be nice to have it.
There is react board components like: https://github.com/ohtomi/react-whiteboard
Maybe we should consider to add priority.

**Comment by bpcurse** [comment]
FTR Very nice and constantly enhanced whiteboard based on node.js:
https://github.com/cracker0dks/whiteboard
